README.md

```diff
@@ -1,56 +1,49 @@
----
-title: Gatsby Code Profile 
```

Comment:
We can reintroduce this when we add a docs page for the profile, but we only need one heading (and the title on the Gatsby docs produces an h1).